tests/test_exprs.py

```diff
@@ -864,42 +864,57 @@ def test_similarity2(self, img_tbl: catalog.Table) -> None:
         with pytest.raises(AttributeError):
             _ = t[t.img.nearest('musical instrument')].show(10)
 
+        t1 = indexed_img_tbl
+        # for a table with a single embedding index, whether we
+        # specify the index or not, the similarity expression
+        # would use that index. So these exressions should be equivalent.
+        sim1 = t1.img.similarity('red truck')
+        sim2 = t1.img.similarity('red truck', idx='img_idx0')
+        assert sim1.id == sim2.id
+        assert sim1.serialize() == sim2.serialize()
+        # index name should be present in the serialized expression
+        assert 'img_idx0' in sim1.serialize()
```

Comment:
That's internal to the implementation and shouldn't be part of the test.

Reply:
Okay, removed.
<!-- Reviewable comment -OEr2XfLB-cnGivWdzAm:b-bl9t1h -->
<!-- Sent from Reviewable.io -->